### PR TITLE
MCKIN-6857: internationalize xblock ooyala

### PR DESCRIPTION
**Pull request by msaqib52**
This PR internationalizes xblock ooyala

### CODE REVIEW

**Comment by msaqib52** [comment]
@naeem91 i have made suggested changes.

File: ooyala_player/templates/html/ooyala_transcript.html [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 <div class="p3sdk-interactive-transcript">
     <div class="p3sdk-interactive-transcript-collapse-visible">
-         <a href="javascript:void(0);" class="p3sdk-interactive-transcript-collapse p3sdk-collapse-onload" label_when_expanded = "&nbsp;<span class='transcript-toggle'>Hide Transcript</span><span class='fa fa-sort-asc transcript-hide fa-2' aria-hidden='true'></span>" label_when_collapsed = "&nbsp;<span class='transcript-toggle'>Show Transcript</span><span class='fa fa-sort-desc transcript-show fa-2' aria-hidden='true'></span>">
-            Hide Transcript
+        {% trans "Hide Transcript" as label_when_expanded %}
+        {% trans "Show Transcript" as label_when_collapsed %}
+        <a href="javascript:void(0);" class="p3sdk-interactive-transcript-collapse p3sdk-collapse-onload" label_when_expanded = "&nbsp;<span class='transcript-toggle'>{{label_when_expanded}}</span><span class='fa fa-sort-asc transcript-hide fa-2' aria-hidden='true'></span>" label_when_collapsed = "&nbsp;<span class='transcript-toggle'>{{label_when_collapsed}}</span><span class='fa fa-sort-desc transcript-show fa-2' aria-hidden='true'></span>">
+            {% trans "Hide Transcript" %}
```
Comment (pomegranited) [inline ooyala_player/templates/html/ooyala_transcript.html]: Could you use the variable `{{ label_when_expanded }}` here?

File: ooyala_player/templates/html/ooyala_player_edit.html [hunk 2]
```diff
@@ -7,95 +7,95 @@
             <label class="label setting-label" for="edit_display_name">Title</label>
```
Comment (pomegranited) [inline ooyala_player/templates/html/ooyala_player_edit.html]: Please mark `Title` for translation.

**Comment by msaqib52** [comment]
@pomegranited i have updated this PR with suggested changes

**Pull request merged**